Helper/Cart.php

```diff
@@ -1386,9 +1386,6 @@ public function getCartData($paymentOnly, $placeOrderPayload, $immutableQuote =
             $taxAmount = 0;
         }
 
-        // include potential rounding difference and reset $diff accumulator
-        $cart['items'][0]['total_amount'] += round($diff);
-        $totalAmount += round($diff);
         $diff = 0;
```

Comment:
This line should go as well. We apply this rounding diff below in this method. Applying it twice seems redundant and could probably cause errors. So all 3 lines above should be removed.

Reply:
@dan-bolt I've updated the code